packages/cudos-cmd/run/run.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs')
+const vm = require('vm')
+const path = require('path')
+
+const { getContractFactory, getContractFromAddress } = require('../../cudos-utilities/contract-services')
+
+global.getContractFactory = getContractFactory
+global.getContractFromAddress = getContractFromAddress
+
+async function runCmd(argv) {
+  if (!fs.existsSync(`${path.resolve('.')}/${argv.scriptFilePath}`)) {
+    console.log(`Script at location ${path.resolve('.')}/${argv.scriptFilePath} does not exist. Execute cudo run --help for more info.`)
+    return
+  }
+  const ds = new vm.Script(fs.readFileSync(argv.scriptFilePath))
+  return await ds.runInThisContext()
```

Comment:
`await` is redundant in `return` statement. Functions are automatically awaited, when returned